### DISCUSSION TITLE
Add differents way to load a model

### DISCUSSION THREAD

**Discussion by ostix360** [ggml-org/llama.cpp#2401] [Ideas]
Hello,
The only way to load a model is by giving a path but in some situation ( in android with an app) a path don't work.
The thing that can be added is a larger way to load models as whisper.cpp does.

**Comment by ggerganov** [comment]
Will probably add support in https://github.com/ggerganov/llama.cpp/pull/2398

**Comment by ostix360** [comment]
ok thanks 
I'll take a look
